script/install_opengrep.py

```diff
@@ -0,0 +1,412 @@
+#!/usr/bin/env vpython3
+# Copyright (c) 2025 The Brave Authors. All rights reserved.
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this file,
+# You can obtain one at https://mozilla.org/MPL/2.0/.
+"""
+Secure Opengrep installer
+
+Downloads opengrep binary directly from GitHub releases and installs to
+tools/opengrep/ directory. Includes optional signature verification using
```

Comment:
all related scripts should be located in `third_party/opengrep` and download should be done into this directory. `tools` is not the right place for this.